data/i18n/bn/bn.toml

```diff
@@ -325,25 +425,100 @@ other = "এই নিবন্ধটি এক বছরেরও বেশি
 other = "প্যাচ রিলিজ"
 
 [post_create_issue]
-other = "একটি ইশূ তৈরি করুন"
+other = "একটি ইস্যু তৈরি করুন"
 
 [prerequisites_heading]
 other = "শুরু করার আগে"
 
 [previous_patches]
 other = "প্যাচ রিলিজ:"
 
+# The following text is displayed when JavaScript isn't available.
+[release_binary_alternate_links]
+other  = """You can find links to download Kubernetes components (and their checksums) in the [CHANGELOG](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG) files.
+Alternately, use [downloadkubernetes.com](https://www.downloadkubernetes.com/) to filter by version and architecture."""
+
+[release_binary_arch]
+other  = "আর্কিটেকচার"
+
+[release_binary_arch_option]
+other = "আর্কিটেকচারগুলো"
+
+[release_binary_copy_link]
+other = "কপি লিংক"
+
+[release_binary_copy_link_certifcate]
+other = "cert"
+
+[release_binary_copy_link_checksum]
+other = "checksum"
```

Comment:
Is there a word for “checksum” in Bangla?

Reply:
There is no actual word but we can use Banlish (english word but typed in bangla ) 

"চেকসাম"